docker/service.Dockerfile

```diff
@@ -15,7 +15,7 @@
 # and updates the entrypoint of the new service container
 #
 ###############################################################################
-FROM python:3.11
+FROM python:3.12
```

Comment:
Nice - I'm tempted to suggest releasing a version of HyBIG with just this change of Python version. Feels like it could be a fair candidate for a patch version.

Reply:
I really don't want to because it's a bunch of overhead for something that will be replaced as soon as I finish all that overhead.  The next release will change functionality significantly and require updated regression-tests.   

I could release it and not deploy it, but that also seems kind of dumb in that you've just created another version that is lying around. 

I just did this in a separate PR to make it easier for someone to review and test.  Since this does run against the existing regressions.

Reply:
Fair enough. Given there isn't a dedicated ticket specifically for this clean up, I can buy that.